src/components/AuctionPriceBar/index.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react'
 
-import { Balance, TokenCode } from 'types'
+import { Balance, DefaultTokenObject } from 'types'
 
 export interface AuctionPriceBarProps {
-  buyToken: TokenCode,
+  buyToken: DefaultTokenObject,
   header: string,
-  sellToken: TokenCode,
+  sellToken: DefaultTokenObject,
   sellTokenPrice: Balance,
 }
 
 const AuctionPriceBar: React.SFC<AuctionPriceBarProps> = ({ sellToken, sellTokenPrice, buyToken, header }) => 
   <div className="auctionLastPrice">
     <small>{`${header} of last auction:`}</small>
-    <big>{`1 ${sellToken} = ${sellTokenPrice} ${buyToken}`}</big>
+    <big>{`1 ${sellToken.symbol || sellToken.name || sellToken.address} = ${sellTokenPrice} ${buyToken.symbol || buyToken.name || buyToken.address}`}</big>
```

Comment:
could consider passing these as a prop `sellTokenName = buyToken.symbol || buyToken.name || buyToken.address` or similar

Reply:
Good idea, won't be able to do it everywhere as sometimes the whole object is necessary, but will wherever possible.